rtmbot/bin/run_rtmbot.py

```diff
@@ -18,11 +19,16 @@ def parse_args():
     )
     return parser.parse_args()
 
-# load args with config path
-args = parse_args()
-config = yaml.load(open(args.config or 'rtmbot.conf', 'r'))
-bot = client.init(config)
-try:
-    bot.start()
-except KeyboardInterrupt:
-    sys.exit(0)
+
+def main(args=None):
```

Comment:
Tiny nitpick (or maybe I'm not understanding something?) -- how is `args` to be used? I see in the `if __name__ == "__main__":` block below that no arguments are passed here.